pkg/api/mlflow/config/config.go

```diff
@@ -42,25 +45,50 @@ func NewServiceConfig() *ServiceConfig {
 }
 
 // Validate validates service configuration.
-func (c ServiceConfig) Validate() error {
-	if err := c.validateArtifactRoot(); err != nil {
-		return eris.Wrap(err, "error validating service configuration")
+func (c *ServiceConfig) Validate() error {
+	if err := c.validateConfiguration(); err != nil {
+		return eris.Wrap(err, `error validating service configuration`)
```

Comment:
we don't need backticks here and in a few other places
```suggestion
		return eris.Wrap(err, "error validating service configuration")
```

Reply:
ok.